checker/raw_result.go

```diff
@@ -111,3 +118,10 @@ type File struct {
 	Type    FileType // Type of file.
 	// TODO: add hash.
 }
+
+type Vulnerability struct {
+	// For OSV: OSV-2020-484
+	// For CVE: CVE-2022-23945
+	ID string
+	// TODO: additional information
```

Comment:
Could we use something like `TODO(vuln)`, so specific TODOs are easier to group and tackle later?

Same with the other TODOs below.

Reply:
I don't specific fields I would like to add at the moment. I just wrote TODO as a placeholder if we want to add extra fields in the future. I'll update the comment to `// Note: add new fields here if needed`. wdut?

Reply:
Meaning something like:

`TODO(vuln): Add additional fields, if needed`

You may know what needs to be done, but consider the following case:
- We want to scrub the TODOs in the repo, and also:
- Allow a newer contributor to work on them
- "Contributor 1, please search for the tasks listed `TODO(vuln)` and work on them"
- "Contributor 2, please search for the tasks listed `TODO(log)` and work on them"
- ...

will allow us to federate things out when needed.

Reply:
done, thanks.